pkg/filesystem/fs/fs.go

```diff
@@ -113,6 +114,10 @@ func NewFileSystem(ctx context.Context, opt ...NewFSOpt) (*Filesystem, error) {
 		if err := d.WaitUntilState(types.DaemonStateRunning); err != nil {
 			return nil, errors.Wrapf(err, "wait for daemon %s", d.ID())
 		}
+		if err := exporter.ExportNydusdEventMetric(d.States.ID, string(types.DaemonStateRunning)); err != nil {
+			log.L.Errorf("export nydusd event metric failed, daemon ID: %s, event: %s, error: %v", d.States.ID, string(types.DaemonStateRunning), err)
+		}
```

Comment:
How about recording nydusd lifetime event in function `WaitUntilState` thus not calling it right after calling `WaitUntilState`

Reply:
It sounds great, but we cannot get the `daemonID`. Maybe we should add a parameter to it or not export the `daemonID`?